.travis.yml

```diff
@@ -1,13 +1,42 @@
 language: generic
 dist: trusty
 
+# Prevent duplicate builds on tag pushes.
+# See https://github.com/mockito/mockito/commit/500519aac4cc13770aa47c1eb7d28e905c27e7aa
+branches:
+  except:
+    - /^release.*/
+    - /^latest-snapshot.*/
+
 services: 
   - docker
 
 before_install: 
   - sudo apt-get install dc
+  - sudo pip install docker-release
+  - docker-release --version
   # prime the cache by fetching the latest release of all images
   - ls teradatalabs/ | awk '{print "teradatalabs/" $0 }' | xargs -L1 time docker pull || true
 
 install: make
 
+before_deploy:
+  # decrypt the github deploy key
+  - openssl aes-256-cbc -K $encrypted_cd2beb64619c_key -iv $encrypted_cd2beb64619c_iv
+    -in .travis/github_tags_push_deploy_key.enc -out .travis/github_tags_push_deploy_key -d
+  - chmod 600 .travis/github_tags_push_deploy_key
+  - eval "$(ssh-agent -s)"
+  - ssh-add .travis/github_tags_push_deploy_key
+  - docker login --username=${DOCKERHUB_LOGIN} --password=${DOCKERHUB_PASSWORD}
+  # reset branch master to this build's commit
+  - git branch -D master || true
```

Comment:
Let me know when https://github.com/kokosing/docker-release/pull/9 is done then :)